### ISSUE TITLE
BIP-2: PVP Tournament

### ISSUE THREAD

**Issue by SotaTek-AnNguyen** [bunicorndefi/BIPs#2]
# Abstract
The `daily PVP` (#1) will be the free & regular activity of the user. It's the first step for players to get familiar with new pvp battle system. There will be another big event for the most mighty players to participate in. The winners will get the huge rewards.

# Specifications
- TBD

**Comment by trivia218** [comment]
Integrate group battles in PVP Tournament on top of the individual battles. Maybe 5-7 people per group, point system based. Match ups will be random. Require BUR entry fee. Maybe earn honor as consolation for those who join, consolation prizes for top 5 and BUR prize from the pool for the top 3.

**Comment by MIN4447** [comment]
There should be individual and team matches.
I hope the individual competition will be a content that everyone can enjoy together from the round of 128 after the preliminary round.
The betting system should also be opened from the round of 128. It will play a good role if users of similar struggle meet.
There will be a lot of burrs in the final. You can incinerate some of them.

A team event is to make users team up. I think it's necessary for the development of the community. It's worth trying before the guild system comes out later. One trainer and one Bernicorn team up with four users in one pair. This will require a strong trainer and give you a reason to raise the level.
Many users may not be able to participate here, so there may be no need for a preliminary round. Then, we will start with betting from the round of 128. If you team up, you can get more favorable responses because the calculation becomes complicated depending on the Bernicorn of various attributes and the order of attack. This also incineration of some of the betting amount.

**Comment by Chubby-Chocobo** [comment]
> Integrate group battles in PVP Tournament on top of the individual battles. Maybe 5-7 people per group, point system based. Match ups will be random. Require BUR entry fee. Maybe earn honor as consolation for those who join, consolation prizes for top 5 and BUR prize from the pool for the top 3.

We'll implement PvP Tournament first. Then the GvG will be the next one. Please stay tuned and wait for our next announcement.

**Comment by Chubby-Chocobo** [comment]
> There should be individual and team matches. I hope the individual competition will be a content that everyone can enjoy together from the round of 128 after the preliminary round.
> 
> The preliminary round will be held for three days and the finals will be held for four days. Everyone can enjoy it for a week. For those who are coming up to the finals, The betting system should also be opened from the round of 128. It will play a good role if users of similar struggle meet. Some of the bets in each fight are incinerated and some of the winners take them. Most of the players who win the bet take it according to the percentage.
> 
> There will be a lot of burrs in the final. In the final, please give us more burrs than the gathered burrs. Those who win the bet should get more burrs, and those who win should also get a separate Bunny.
> 
> A team event is to make users team up. I think it's necessary for the development of the community. It's worth trying before the guild system comes out later. One trainer and one Bernicorn team up with four users in one pair. This will require a strong trainer and give you a reason to raise the level. Many users may not be able to participate here, so there may be no need for a preliminary round. Then, we will start with betting from the round of 128. If you team up, you can get more favorable responses because the calculation becomes complicated depending on the Bernicorn of various attributes and the order of attack. This also incineration of some of the betting amount.
> 
> The team event will be held once every two to four weeks, and it may be better to raise more prize money.

Yup. We'll setup like other e-sport leagues & tournaments. Please stay tuned and we'll reveal more detailed about this.
